frontend/app/src/components/display/DateDisplay.vue

```diff
@@ -57,35 +55,21 @@ const showTooltip = computed(() => {
 const splittedByMillisecondsPart = computed(() =>
   get(formattedDate).split('.')
 );
+
+const { copy, copied } = useCopy(formattedDate);
 </script>
 
 <template>
-  <span>
-    <RuiTooltip
-      :popper="{ placement: 'top' }"
-      open-delay="400"
-      :disabled="!showTooltip"
+  <span :class="{ blur: !shouldShowAmount }" @click="copy()">
```

Comment:
So `blur` works fine on `CopyTooltip` but click is not propagated from `RuiTooltip` which is probably because we didn't expose the listeners too. I will keep the PR open a bit until the release and try to see if adjusting the Tooltip would help here, so that we can skip the extra span.

Reply:
https://github.com/rotki/ui-library-compat/issues/59
need this to be deployed